charts/restate-helm/README.md

```diff
@@ -7,3 +7,6 @@ Helm chart for Restate as a single-node StatefulSet.
 ```bash
 helm install restate oci://ghcr.io/restatedev/restate-helm --namespace restate --create-namespace
 ```
+
+# Running a replicated cluster
+You can find example values for a 3-node replicated cluster in [replicated-values.yaml](./replicated-values.yaml)
```

Comment:
Being a helm noob, how would I used these values to deploy the three node cluster?

Reply:
added an example to the readme. btw, if you want to see it work locally, you can enable the orbstack kubernetes feature, and do:
```
helm install -f ./charts/restate-helm/replicated-values.yaml  restate ./charts/restate-helm
```

Reply:
Nice, I'll give it a try :-)